### PR TITLE
feat: Edition slice tile - Tile T

### PR DESCRIPTION
**Pull request by jimhunty**
Tile T is like Tile D, but with a different ratio image.

screenshot 2019-02-18 at 14 22 40

### CODE REVIEW

File: packages/edition-slices/src/tiles/tile-t/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import { TileImage, TileLink, TileSummary } from "../shared";
+import styles from "./styles";
+
+const TileT = ({ onPress, tile }) => (
+  <TileLink onPress={onPress} tile={tile}>
+    <View style={styles.container}>
+      <TileImage
+        aspectRatio={16 / 9}
+        style={styles.imageContainer}
+        uri={tile.article.leadAsset.crop169.url}
```
Comment (tuncaulubilge) [inline packages/edition-slices/src/tiles/tile-t/index.js]: Can you use the shared "getCrop" function from tiles/shared for this url? (it was just merged couple of mins ago in another PR)

**Comment by aronshamash** [comment]
@jimhunty needs a rebase

File: packages/edition-slices/src/tiles/tile-t/index.js (hunk 1 above)
Comment (jimhunty) [reply packages/edition-slices/src/tiles/tile-t/index.js]: Should be done now.

File: packages/edition-slices/edition-tiles.showcase.js [hunk 2]
```diff
@@ -16,7 +16,11 @@ import {
   TileM,
   TileN,
   TileO,
+<<<<<<< HEAD
```
Comment (tuncaulubilge) [inline packages/edition-slices/edition-tiles.showcase.js]: ☝️

Comment (jimhunty) [reply packages/edition-slices/edition-tiles.showcase.js]: 😱

**Pull request merged**